pkg/controller/node/node_controller.go

```diff
@@ -1280,3 +1426,11 @@ func getErrorString(err error) string {
 	}
 	return ""
 }
+
+func (ctrl *Controller) IsLayeredPool(pool *mcfgv1.MachineConfigPool, mosc *mcfgv1alpha1.MachineOSConfig, mosb *mcfgv1alpha1.MachineOSBuild) bool {
+	fg, err := ctrl.fgAcessor.CurrentFeatureGates()
```

Comment:
check for FGs, GetConfigAndBuild might need this

Reply:
@cdoern please check validity of this function

Reply:
fine for now, the reason this is so simple is because we need to call `GetConfigAndBuild` outside of `IsLayeredPool` bc the config and build need to be used outside of the func. Could move the call for `GetConfigAndBuild` into `IsLayeredPool` and return the values, but I think this is unnecessary for now.